configure.ac

```diff
@@ -444,18 +444,28 @@ AC_ARG_ENABLE(ncp-app,
             enable_ncp_app=${enableval}
             enable_ncp_app_mtd=${enableval}
             enable_ncp_app_ftd=${enableval}
+            enable_ncp_app_radio=${enableval}
```

Comment:
I'm not sure if there is any use-case. I think enabling *raw link api* in *ftd*/*mtd* mode allows a single firmware providing different layer APIs.